apps/studio/components/to-be-cleaned/Integration/Vercel.utils.ts

```diff
@@ -1,59 +1,49 @@
 import { post, get } from 'lib/common/fetch'
 import { INTEGRATION_ENVS_ALIAS } from 'lib/vercelConfigs'
-import { Dictionary } from 'types'
 import { API_URL } from 'lib/constants'
 
-export async function fetchVercelProjects({
-  vercelTeamId,
+async function doFetchVercelProjects({
```

Comment:
side note: this can probably be shifted to a react query to make things easier - but not necessary in this PR